strictdoc/export/html/form_objects/grammar_form_object.py

```diff
@@ -28,22 +28,33 @@ def __init__(
         field_mid: str,
         field_name: str,
         document_mid: str,
+        field_human_title: Optional[str] = None,
```

Comment:
Let's remove the default `= None` part.

(StrictDoc's code style is rather verbose with the added value of a maximum transparency. The default argument values are almost never used.)